### DISCUSSION TITLE
How do I access other paths?

### DISCUSSION THREAD

**Discussion by mvajid** [filebrowser/filebrowser#3268] [General]
How do I access other directories on my server? I see only the /usr/local/bin directory with filebrowser and filebrowser.db files. I am not running this in a docker. I read the documentation but can't figure it out. Thanks.

PS. I tried looking for other discussions but didn't find anything. I want to add /home/user/ path for my user (and other users). Tried adding that in Rules> Allow but I still dont see how I can access other paths in the file browser.

**Comment by mvajid** [comment]
Never mind, I figured it out. It was taking /home/home/user so I had to specify just /user.
Also, there was a permission problem with a folder that is mounted using rclone.

**Reply by loadship** [reply]
How to deal with this problem
